src/Generator/EpubGenerator.php

```diff
@@ -398,17 +403,23 @@ private function getXhtmlTitle( Book $book ) {
 	}
 
 	private function getXhtmlAbout( Book $book, $wsUrl ) {
-		$list = '<ul>';
-		$listBot = '<ul>';
-		foreach ( $book->credits as $name => $value ) {
-			if ( $value[ 'bot' ] !== null ) {
-				$listBot .= '<li>' . htmlspecialchars( $name, ENT_QUOTES ) . "</li>\n";
-			} else {
-				$list .= '<li>' . htmlspecialchars( $name, ENT_QUOTES ) . "</li>\n";
+		if ( !$book->options['credits'] ) {
+			$list = $this->intuition->msg( 'credits-default-message' ) ??
```

Comment:
The language of Intuition should be set before this, so that it uses the book's language rather than the interface language.

Reply:
Good point! Done.